build-system/tasks/bundle-size.js

```diff
@@ -110,72 +103,33 @@ async function storeBundleSize() {
     return;
   }
 
-  if (!process.env.GITHUB_ARTIFACTS_RW_TOKEN) {
-    log(
-      red(
-        'ERROR: Missing GITHUB_ARTIFACTS_RW_TOKEN, cannot store the ' +
-          'bundle size in the artifacts repository on GitHub!'
-      )
-    );
-    process.exitCode = 1;
-    return;
-  }
-
-  const octokit = new Octokit({
-    auth: `token ${process.env.GITHUB_ARTIFACTS_RW_TOKEN}`,
-  });
-
-  for (const [compression, extension, getBundleSize] of [
-    ['gzip', '', getGzippedBundleSize],
-    ['brotli', '.br', getBrotliBundleSize],
-  ]) {
-    const bundleSize = `${getBundleSize()}KB`;
-    const bundleSizeFile = `${gitCommitHash()}${extension}`;
-    const githubApiCallOptions = Object.assign(buildArtifactsRepoOptions, {
-      path: path.join('bundle-size', bundleSizeFile),
+  const commitHash = gitCommitHash();
+  try {
+    const response = await requestPost({
+      uri: url.resolve(
+        bundleSizeAppBaseUrl,
+        path.join('commit', commitHash, 'report')
+      ),
+      json: true,
+      body: {
+        token: process.env.BUNDLE_SIZE_TOKEN,
+        // TODO(#21275): replace the gzippedBundleSize value once the
+        // bundle-size app prefers Brotli.
+        gzippedBundleSize: `${getGzippedBundleSize()}KB`,
+        brotliBundleSize: `${getBrotliBundleSize()}KB`,
+      },
     });
-
-    try {
-      await octokit.repos.getContents(githubApiCallOptions);
-      log(
-        'The file',
-        cyan(`bundle-size/${bundleSizeFile}`),
-        'already exists in the',
-        'build artifacts repository on GitHub. Skipping...'
+    if (response.statusCode < 200 || response.statusCode >= 300) {
+      throw new Error(
+        `${response.statusCode} ${response.statusMessage}: ` + response.body
       );
-      continue;
-    } catch {
-      // The file was not found in the GitHub repository, so continue to create
-      // it...
-    }
-
-    try {
-      await octokit.repos.createOrUpdateFile(
-        Object.assign(githubApiCallOptions, {
-          message: `bundle-size: ${bundleSizeFile} (${bundleSize})`,
-          content: Buffer.from(bundleSize).toString('base64'),
-        })
-      );
-      log(
-        'Stored the new',
-        cyan(compression),
-        'bundle size of',
-        cyan(bundleSize),
-        'in the artifacts',
-        'repository on GitHub'
-      );
-    } catch (error) {
-      log(
-        red(
-          `ERROR: Failed to create the bundle-size/${bundleSizeFile} file in`
-        ),
-        red('the build artifacts repository on GitHub!')
-      );
-      log(red('Error message was:'), error.message);
-      return Promise.reject(error);
     }
+  } catch (error) {
```

Comment:
Does this try/catch catch any exceptions besides the one you throw right above? Because it feels a little weird using a try/catch if you're the only one throwing exceptions.

Reply:
Yeah, it could catch an error from `requestPost`